src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -7339,6 +7339,78 @@ void CodeGen::genArm64EmitterUnitTestsSve()
                               INS_SCALABLE_OPTS_UNPREDICATED);
     theEmitter->emitIns_R_R_I(INS_sve_str, EA_SCALABLE, REG_V2, REG_R3, 255, INS_OPTS_NONE,
                               INS_SCALABLE_OPTS_UNPREDICATED);
+
+#ifdef ALL_ARM64_EMITTER_UNIT_TESTS_SVE_UNSUPPORTED
+    // IF_SVE_GG_3A
+    // LUTI2 <Zd>.B, {<Zn>.B }, <Zm>[<index>]
+    // luti2   z0.b, {z0.b}, z0[0]  // 01000101-00100000-10110000-00000000
+    // CHECK-INST: luti2   z0.b, { z0.b }, z0[0]
+    // CHECK-ENCODING: [0x00,0xb0,0x20,0x45]
```

Comment:
I like this idea. @a74nh and I have implemented several encodings at this point that capstone cannot decode, so maybe as a cleanup step, we should come back to all the unsupported encodings, and add this validation in.

Reply:
> as a cleanup step, we should come back to all the unsupported encodings, and add this validation in.

This would be useful. Plus, remove all the unreached asserts. I recommend doing that after all the other instructions have been completed.

Reply:
This actually isn't checked or validated. It's just meant for reference. I copy-pasted the test from LLVM because I didn't want to type it out by hand in-case I made a mistake. Sorry for the confusion.

It would be nice to have a way to verify it like LLVM, but that kind of work is out-of-scope.